lib/Comment/styles.scss

```diff
@@ -0,0 +1,56 @@
+.conversation__text__cutoff {
+  &:after {
+    @apply absolute h-8 w-full;
+    content: ' ';
+    left: 40px;
+    top: 96px;
+    background: linear-gradient(
+      to bottom,
+      rgba(255, 255, 255, 0) 0%,
+      rgba(253, 253, 253, 0.5) 50%,
+      #fff 100%
+    );
+  }
+}
+
+.conversation__inactive {
+  &:hover {
+    .conversation__text__cutoff {
+      &:after {
```

Comment:
As for the `after` pseudo selector, in a perfect world we would factor this to not use `after`. To be fair, after is a bit of a smell. It's sort of encroaching on Reacts territory of displaying rendering things based on conditions.

Again, out of scope though

Reply:
I agree with that, and I had done it in other places. I will probably do the same for these now I'm looking at it